source/docs/appendices/migration-guides.md

```diff
@@ -9,13 +9,22 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.8.0` -> `2.9.0`
 
-No action needed for migrating to this version.
+### Customs options `price` and `price_type` field deprecations
+
+As of the version 2.9.0, the `price` and `price_type` fields of the types `ProductCustomOption` and `ProductCustomOptionValue` have been deprecated in order to provide both the price including and excluding taxes for each options. Custom options prices or rate are now carried by the `extraCost` field available in those types. To stop using those deprecated fields, the following files have been modified:
+
+- `theme/modules/Cart/CartItem/CartItemOptionsUpdater/CartItemOptionsUpdaterFragment.gql`
+- `theme/components/organisms/Configurator/ProductConfiguratorFragment.gql`
+- `theme/components/organisms/Configurator/CustomOption/CustomOptionLabelWithPrice.js`
+- `theme/modules/ProductView/withSelectedCustomOptions.js`
+
+So if you have overridden those files, you have to apply [the same update than the one done in Front-Commerce](https://gitlab.com/front-commerce/front-commerce/-/commit/79c69a9d294dd58979dbd5d94153edf9e29b3d28?merge_request_iid=622).
```

Comment:
```suggestion
So if you have overridden those files, you have to apply [the same update than the one done in Front-Commerce](https://gitlab.com/front-commerce/front-commerce/-/commit/79c69a9d294dd58979dbd5d94153edf9e29b3d28?merge_request_iid=622).

We highly recommend you to test your product and cart page for products with custom options of both "fixed price" and "percentage" types if your projects makes use of this Magento feature.
```

I feel it's good to insist on this one after reading the code changes related to this feature. We're never too safe.

Reply:
done